guides/common/modules/proc_installing-convert2rhel.adoc

```diff
@@ -0,0 +1,27 @@
+[id="installing-convert2rhel_{context}"]
+
+= Installing Convert2RHEL
+
+The following procedure makes Convert2RHEL available to your system.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Content* *>* *Products* and click *Create Product*.
+. In the *Name* field, enter the name of the product.
+{Project} automatically completes the *Label* field based on your entry for *Name*.
+. Optional: From the *GPG Key* list, select the GPG key for the product.
+. Optional: From the *SSL CA Cert* list, select the SSL CA certificate for the product.
+. You do not need to complete *SSL Client Cert* or *SSL Client Key*.
```

Comment:
This is not an instruction. Maybe it's even superfluous, because both instructions in lines 11 and 12 are "optional".